src/repart/repart.c

```diff
@@ -2457,6 +2459,37 @@ static int config_parse_encrypted_volume(
         return 0;
 }
 
+static int config_parse_tpm2_pcrs(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Partition *partition = ASSERT_PTR(data);
+        int r;
+
+        assert(rvalue);
+
+        if (isempty(rvalue)) {
+                /* Clear existing PCR values if empty */
+                partition->tpm2_hash_pcr_values = mfree(partition->tpm2_hash_pcr_values);
+                partition->tpm2_n_hash_pcr_values = 0;
+                return 0;
+        }
+
+        r = tpm2_parse_pcr_argument_append(rvalue, &partition->tpm2_hash_pcr_values, &partition->tpm2_n_hash_pcr_values);
```

Comment:
Line-break please.

Maybe drop `r` and just `return tpm2_parse_pcr_argument_append(…)`?

Reply:
I am using `r` because the other cases are doing the same, so for consistency better to leave it as it is

Reply:
Ill add the line break ofc

Reply:
I'd also like `return tpm2_parse_pcr_argument_append(...)`.